src/snowflake/snowpark/modin/pandas/dataframe.py

```diff
@@ -2265,7 +2287,15 @@ def shift(
     def set_index(
         self,
         keys: IndexLabel
-        | list[IndexLabel | pd.Index | pd.Series | list | np.ndarray | Iterable],
+        | list[
+            IndexLabel
+            | pd.Index
+            | pandas.Index
```

Comment:
we should not allow pandas Index as an input.

Reply:
Fixed

Reply:
I see it here?

Reply:
weird, I swear I removed it